src/player_avatar.c

```diff
@@ -884,22 +884,26 @@ int PlayerAvatar_DistortionWorldState (PlayerAvatar * const playerAvatar)
     return state;
 }
 
-BOOL sub_0205F158 (PlayerAvatar * const playerAvatar)
+// these functions can be simplified, but they'll no longer match
+
+BOOL PlayerAvatar_DistortionGravityChanged (PlayerAvatar * const playerAvatar)
 {
-    int v0 = PlayerAvatar_DistortionWorldState(playerAvatar);
+    int state = PlayerAvatar_DistortionState(playerAvatar);
 
-    if ((v0 == 0) || (v0 == 1)) {
+    if (state == AVATAR_DISTORTION_STATE_NONE || state == AVATAR_DISTORTION_STATE_ACTIVE) {
         return FALSE;
     }
 
     return TRUE;
 }
 
-BOOL sub_0205F16C (PlayerAvatar * const playerAvatar)
+BOOL PlayerAvatar_DistortionStateOnFloor (PlayerAvatar * const playerAvatar)
 {
-    int v0 = PlayerAvatar_DistortionWorldState(playerAvatar);
+    int state = PlayerAvatar_DistortionState(playerAvatar);
 
-    if ((v0 == 0) || (v0 == 1) || (v0 == 2)) {
+    if (state == AVATAR_DISTORTION_STATE_NONE 
```

Comment:
```c
return state == AVATAR_DISTORTION_STATE_NONE
    || state == AVATAR_DISTORTION_STATE_ACTIVE
    || state == AVATAR_DISTORTION_STATE_FLOOR;
```

Reply:
Does it really not match with this change? Also, why remove this indentation in the most recent revision?

Reply:
My monkey brain read right past the `return` in your comment and read it as a suggestion to change the indentation there. Not my finest moment.